src/style.css

```
@@ -0,0 +1,41 @@
+html {
+  --bg-active: url(images/star-active.svg);
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :first-child {
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group selectors